src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -584,6 +585,7 @@ class ArnoldAttributes : public IECoreScenePreview::Renderer::AttributesInterfac
 
 			m_linkedLights = attribute<IECore::StringVectorData>( g_linkedLights, attributes );
 
+			m_sssSetName = NULL;
```

Comment:
This is doubly redundant. intrusive_ptr initialises to null by default anyway, and it's also explicitly initialised to null in the initialiser list above.